src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -277,90 +305,132 @@ const WorkspaceDashboard = _.flow(
       ])
     ]),
     div({ style: Style.dashboard.rightBox }, [
-      div({ style: Style.dashboard.header }, ['Workspace information']),
-      div({ style: { display: 'flex', flexWrap: 'wrap', margin: -4 } }, [
-        h(InfoTile, { title: 'Creation date' }, [new Date(createdDate).toLocaleDateString()]),
-        h(InfoTile, { title: 'Last updated' }, [new Date(lastModified).toLocaleDateString()]),
-        h(InfoTile, { title: 'Submissions' }, [submissionsCount]),
-        h(InfoTile, { title: 'Access level' }, [roleString[accessLevel]]),
-        Utils.canWrite(accessLevel) && h(InfoTile, { title: 'Est. $/month' }, [
-          storageCostEstimate || '$ ...'
-        ]),
-        h(InfoTile, { title: 'Google Project Id' }, [
-          div({ style: { display: 'flex' } }, [
-            h(TooltipCell, [googleProject]),
-            h(ClipboardButton, { text: googleProject, style: { marginLeft: '0.25rem' } })
+      div({ style: { paddingTop: '1rem' } }, [
+        div({ style: Style.dashboard.rightBoxContainer }, [
+          h(Collapse, {
+            title: h2({ style: Style.dashboard.collapsableHeader }, ['Workspace information']),
+            initialOpenState: workspaceInfoPanelOpen !== undefined ? workspaceInfoPanelOpen : true,
+            titleFirst: true,
+            onClick: () => setWorkspaceInfoPanelOpen(workspaceInfoPanelOpen === undefined ? false : !workspaceInfoPanelOpen)
+          }, [
```

Comment:
It looks like this is repeated for each section. Could this be factored out into a component?

Reply:
Most definitely, I will take care of that